tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
+
+
+@pytest.mark.usefixtures('init_store')
+def test_restore_run(init_store):
+    init_store.restore_run("d57a45f3763e4827b7c03f03d60dbbe1")
+    run_restored = init_store.get_run("d57a45f3763e4827b7c03f03d60dbbe1")
+    assert run_restored._info._lifecycle_stage == LifecycleStage.ACTIVE
+
+
+@pytest.mark.usefixtures('init_store')
+def test_restore_run_of_active_run(init_store):
+    with pytest.raises(MlflowException) as excinfo:
+        init_store.restore_run("d57a45f3763e4827b7c03f03d60dbbe1")
+        assert "must be in the 'deleted' state" in str(excinfo.value)
+
+
+@pytest.mark.usefixtures('init_store')
+def test_log_metric(init_store):
+    new_metric = Metric(key="new_metric", value=7.0, timestamp=10, step=0)
+    expected_elastic_metrics = [ElasticMetric(key="metric0", value=15.0,
+                                              timestamp=1597324762700, step=0, is_nan=False),
+                                ElasticMetric(key="metric0", value=7.0,
+                                              timestamp=1597324762742, step=1, is_nan=False),
+                                ElasticMetric(key="metric0", value=20.0,
+                                              timestamp=1597324762778, step=2, is_nan=False),
+                                ElasticMetric(key="metric1", value=20.0,
+                                              timestamp=1597324762815, step=0, is_nan=False),
+                                ElasticMetric(key="metric1", value=0.0,
+                                              timestamp=1597324762847, step=1, is_nan=False),
+                                ElasticMetric(key="metric1", value=7.0,
+                                              timestamp=1597324762890, step=2, is_nan=False),
+                                ElasticMetric(key="new_metric", value=7.0,
+                                              timestamp=10, step=0, is_nan=False)]
+    expected_elastic_latest_metrics = [ElasticLatestMetric(key="metric0", value=20.0,
+                                                           timestamp=1597324762778, step=2,
+                                                           is_nan=False),
+                                       ElasticLatestMetric(key="metric1", value=7.0,
+                                                           timestamp=1597324762890, step=2,
+                                                           is_nan=False),
+                                       ElasticLatestMetric(key="new_metric", value=7.0,
+                                                           timestamp=10, step=0, is_nan=False)]
+    init_store.log_metric("7b2e71956f3d4c08b042624a8d83700d", new_metric)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, latest_metric in enumerate(actual_run.latest_metrics):
+        assert latest_metric == expected_elastic_latest_metrics[i]
+    for i, metric in enumerate(actual_run.metrics):
+        assert metric == expected_elastic_metrics[i]
```

Comment:
I'm not sure to understand these checks, here you logged a single metric, so you should only check that this metric is present in the run data